tests/foreman/api/test_multiple_paths.py

```diff
@@ -116,14 +117,16 @@ class EntityTestCase(APITestCase):
         entities.Architecture,
         entities.AuthSourceLDAP,
         entities.ComputeProfile,
-        entities.ComputeResource,
+        entities.AbstractComputeResource,
```

Comment:
I think that `AbstractComputeResource` can be dropped from `tests/foreman/api/test_multiple_paths.py` completely. The `AbstractComputeResource` class doesn't really correspond to anything on the server, and it exists just to make the definitions of `LibvirtComputeResource`, `DockerComputeResource` and so on nicer.

Also, the usage of "abstract" classes is what forced us in to using partials in the first place. If we can drop partials, then this module will become a little less complex and easier to work with.